src/pipx/commands/inject.py

```diff
@@ -55,6 +55,19 @@ def inject_dep(
             pip_args=pip_args,
             verbose=verbose,
         )
+
+    if not force and venv.has_package(package_name):
```

Comment:
We might as well test whether `Venv.list_installed_packages()` contains the package. The added function looks fine, though.

Reply:
I noticed the `list_installed_packages()` but it is implemented by run a pip command as subprocess, so I did not reuse it.